src/domain/Tokens.java

```diff
@@ -0,0 +1,76 @@
+package domain;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+
+public class Tokens {
+
+    private ArrayList<String> reservedWords;
+    private ArrayList<String> separators;
+    private ArrayList<String> operators;
+
+
+    public Tokens() {
+        reservedWords=new ArrayList<>();
+        separators=new ArrayList<>();
+        operators= new ArrayList<>();
+        read();
+    }
+
+    public void read(){
+        String fileName = "Tokens.in";
+        String line;
+        int i=1;
+        try {
+            FileReader fileReader =
+                    new FileReader(fileName);
+            BufferedReader bufferedReader =
+                    new BufferedReader(fileReader);
+
+            while((line = bufferedReader.readLine()) != null) {
+                if(i <= 16) {
+                    operators.add(line.strip());
+                }
+                else if(i > 16 && i <=22){
+                    if (line.strip().equals("<space>")){
+                        separators.add(" ");
+                    }
+                    else{
+                        separators.add(line.strip());
+                    }
+                }
+                else{
+                    reservedWords.add(line.strip());
+                }
+                i++;
```

Comment:
Very hard to understand what the i represents, maybe rename it or change the format of the tokens.in file to make more sense, for example separate each category by a new line

Every time you change the tokens, you need to change the reading logic as well